src/lib/stores/user.ts

```diff
@@ -0,0 +1,12 @@
+import { writable } from 'svelte/store';
+
+import { fetchUser } from '$lib/services/user-service';
+
+export const user = writable<User>(null);
+
+const loadUser = async (): Promise<void> => {
```

Comment:
Would this be better off just as a single API call here? https://github.com/temporalio/ui/blob/main/src/routes/__layout.svelte#L11-L21

Reply:
done